app/screens/login/login.js

```diff
@@ -299,15 +299,28 @@ class Login extends PureComponent {
                 />
             );
         } else {
+            const additionalStyle = {};
+            if (this.props.config.EmailLoginButtonColor) {
+                additionalStyle.backgroundColor = this.props.config.EmailLoginButtonColor;
```

Comment:
These might need to be marked as experimental.

Also, this is making me wish we had better theme support for the login screen.

Reply:
I can go either way on marking these as experimental, @enahum thoughts?

Reply:
Well as far as I know those are going to be included in the server config so I don’t think we need to mark them

Reply:
Yes, PR for platform is coming to include those server side.